demosplan/DemosPlanCoreBundle/Logic/User/UserHandler.php

```diff
@@ -1752,61 +1752,67 @@ protected function checkMandatoryFieldsOrga($data, $currentOrga)
      *
      * @param string $organisationId Indicates the organisation whose data will be wiped
      *
-     * @return orga|bool The wiped organisation if all operations are successful, otherwise false
+     * @return orga|array The wiped organisation if all operations are successful, otherwise false
```

Comment:
comment has to be adjusted here

Reply:
done in [5c3c6df](https://github.com/demos-europe/demosplan-core/pull/2681/commits/5c3c6df381e845a5949f2d7998d74338942e678a)